SubscribeFunction/SubscribeFunction.py

```diff
@@ -1,17 +1,16 @@
 import os
 import json
 import boto3
+from datetime import datetime
 from botocore.vendored import requests
```

Comment:
You should try to replace the botocore version of requests with just `import requests` since I believe the boto vendored requests is either gone alread or soon to be gone

Reply:
Nvm, you don't even use requests, you can just remove it